go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/getlantern/lantern-client
 
-go 1.22.0
+go 1.22.3
```

Comment:
Would you revert this back to just 1.22? We won't be updating the go minor version in `go.mod`, as discussed in stand-up.